shell.go

```diff
@@ -7,14 +7,19 @@ import (
 	"strings"
 )
 
-// CommandInShell sets up a new command with the environment set up by users environment.
+// stringLiteral is a type of string that cannot be passed as a variable.
+// This helps to avoid code injection by enforcing executable names to be hard coded.
+type stringLiteral string
+
+// CommandInShell sets up a new command with the environment set up by user's environment.
 // In darwin or other Unix systems the shell will be loaded by running the terminal and accessing env command.
-// The command is run directly so stdout and stderr can be read directly from the returnd `exec.Cmd`.
-func CommandInShell(cmd string, args ...string) *exec.Cmd {
+// The command is run directly so stdout and stderr can be read directly from the returned `exec.Cmd`.
+// The cmd parameter must be a string constant to avoid possible code injection attacks.
```

Comment:
Does a string constant work or is it just string literals? If it's the latter, we should probably say literal instead.

Reply:
Constants and literals both work